msp/cost/master_org_cost_policy_currency/master_org_cost_policy_currency.pt

```diff
@@ -0,0 +1,1026 @@
+        } else {
+          seriesData.push(Math.round(tempTotal[0].cost))
+        }
+      })
+      chartDataArray.push(seriesData.join())
+    }
+    var chartData = "chd=a:" + chartDataArray.join('|')
+    var encodedCategories = encodeURIComponent(categories.join('|')).replace(/[(]/gi,'%28').replace(/[)]/gi,'%29');
+    if ( encodedCategories.length < 1) {
+      var chartCategories = "chdl=Unknown"
+    } else {
+      var chartCategories = "chdl=" + encodedCategories
+    }
+    var chartColors = "chco=" + colorArray.slice(0,categories.length).join();
+    var chartXAxis = "chxl=0:|" + stringMonths.join('|')
+    
+    var billingCenters = "All";
+
+    // Flatten filtered orgs
+    var includedOrgs = _.pluck(ds_filtered_user_organizations, 'name').join('; ')
+
+    report = {
+      includedOrgs: includedOrgs
+      currentMonthName: currentMonthName,
+      billingCenters: billingCenters,
+      chartType: "cht=bvs",
+      chartSize: "chs=800x500",
+      chartTitle: "chtt=Spending+Overview",
+      chartAxis: "chxt=x,y",
+      chartXAxis: chartXAxis,
+      chartAxisFormat: "chxs=1N*c" + param_final_currency + "s*"
+      chartData: chartData,
+      chartCategories: chartCategories,
+      chartColors: chartColors,
+      chartExtension: "chof=.png",
+      chartKeyLocation: "chdlp=b",
+      currentSpend:  Math.round(current_month_total),
+      currencyCode: param_final_currency,
+      reportData: ds_report
+    }
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_scheduled_report" do
+  validate $ds_generated_report do
+    summary_template "Multi-Organization Full Cost Scheduled Report"
+    detail_template <<-EOS
+# Full Cost Report for {{ data.currentMonthName }}
+### Included Organizations: {{ data.includedOrgs }}
+
+### Cost Metric: {{ parameters.param_cost_metric }}
+### Chart Dimension: {{ parameters.param_graph_dimension }}
+
+### Currency Converted to: {{ parameters.param_final_currency }}*
+
+![Spending Overview Chart](https://image-charts.com/chart?{{ data.chartType }}&{{ data.chartSize }}&{{ data.chartTitle }}&{{ data.chartAxis }}&{{ data.chartXAxis }}&{{ data.chartAxisFormat }}&{{ data.chartData }}&{{ data.chartCategories }}&{{ data.chartColors }}&{{ data.chartKeyLocation }}&{{ data.chartExtension }} "Spending Overview Chart")
+
+| Range | Daily Average | Previous | Current (incomplete) |
+| ----- | ------------- | -------- | -------------------- |
+{{ range data.reportData -}}
+  | {{ .range }} | {{ .avg }} | {{ .previous}} | {{ .current }} |
+{{ end -}}  
+  
+### For more detailed cost information, visit [Optima](https://analytics.rightscale.com/orgs/{{ rs_org_id }}/dashboard).
+  
+For more information on this report, please view the [README](https://github.com/rightscale/policy_templates/tree/master/cost/master_org_cost_policy_currency).<br>
```

Comment:
This isn't the correct path for the readme

Reply:
@cdwilhelm resolved